simple/csharp/extract-variable_after2.cs

```diff
@@ -0,0 +1,11 @@
+void renderBanner() 
+{
+  boolean mustRdraw = (frame.isChanged() || target.isChanged()) 
+    || experiment.isRunning();
```

Comment:
I'd replace it like this:
boolean isChanged = frame.isChanged || target.isChanged;
boolean mustRedraw = isChanged || experiment.isRunning();